sdk/core/System.ClientModel/src/Message/ServiceResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Primitives;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+public abstract class ServiceResponse
```

Comment:
What is the analogous piece on the request side?
- ClientMessage/ServiceMessage?
- ClientRequest/ServiceResponse?